src/5e-SRD-Features.json

```diff
@@ -48,6 +71,36 @@
     "desc": [
       "Starting at 2nd level, you can throw aside all concern for defense to attack with fierce desperation. When you make your first attack on your turn, you can decide to attack recklessly. Doing so gives you advantage on melee weapon attack rolls using Strength during this turn, but attack rolls against you have advantage until your next turn."
     ],
+    "feature_specific": {
+      "advantages": [
```

Comment:
Is there a way we could wrap this in some kind of "action" structure? So the client knows it can be added to the character's possible list of actions. The action would grant this advantage. Just a thought, maybe one for a future PR though.